tests/Feature/Http/Middleware/AuthenticateTest.php

```diff
@@ -34,7 +34,7 @@ protected function setUp(): void
     public function testUserIsNotRedirectedWhenLoggedIn()
     {
         // Given
-        $user = User::factory()->create();
+        $user = AuthorizedUser::factory()->create();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar $authorizedUser .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create();
````